src/pages/registration/description_tab/DatePickerField.tsx

```diff
@@ -52,6 +55,7 @@ export const DatePickerField = () => {
           updateDateValues(newDate, yearOnly);
           setDate(newDate);
         }}
+        disabled={isApprovedNviCandidate}
```

Comment:
NB! skal ikke være disabled for kurator. En mulighet er å endre `isApprovedNviCandidate` til å bli `disableNviCriticalFields` el?